pkg/portal/portal.go

```diff
@@ -261,21 +265,38 @@ func (p *portal) unauthenticatedRoutes(r *mux.Router) {
 }
 
 func (p *portal) aadAuthenticatedRoutes(r *mux.Router) {
-	for _, name := range AssetNames() {
-		if name == "v1/build/index.html" {
-			r.NewRoute().Methods(http.MethodGet).Path("/").HandlerFunc(p.index)
-			continue
-		}
+	var names []string
 
-		if name == "v2/build/index.html" {
-			r.NewRoute().Methods(http.MethodGet).Path("/v2").HandlerFunc(p.indexV2)
-			continue
+	err := filepath.Walk(".", func(path string, info fs.FileInfo, err error) error {
+		if err != nil {
+			return err
 		}
 
-		fmtName := strings.TrimPrefix(name, "v1/build/")
-		fmtName = strings.TrimPrefix(fmtName, "v2/build/")
+		names = append(names, path)
+		return nil
+	})
+
+	if err != nil {
+		log.Fatal(err)
+	}
 
-		r.NewRoute().Methods(http.MethodGet).Path("/" + fmtName).HandlerFunc(p.serve(name))
+	for _, name := range names {
+		regexp, _ := regexp.Compile(`v[1,2]/build/.*\..*`)
+		name := regexp.FindString(name)
+	nameLoop:
+		switch name {
+		case "v1/build/index.html":
+			r.NewRoute().Methods(http.MethodGet).Path("/").HandlerFunc(p.index)
+		case "v2/build/index.html":
+			r.NewRoute().Methods(http.MethodGet).Path("/").HandlerFunc(p.indexV2)
+		case "":
+			break nameLoop
```

Comment:
why do you break nameloop here? This didn't use to be this logic. please remove it. The break in the `switch` was to be explicitly exiting the `switch` because there was no action performed inside the `case`.

Reply:
@facchettos - Thanks for pointing it out.  Actually, I got  `ineffective break statement. Did you mean to break out of the outer loop?` from go-staticcheck in the IDE if put `break` for the case `""` and the golangci-lint will be failed for that. That's why I put a label `nameLoop` there as a workaround. (yeah, the naming is not that clear, sorry) 

Anyway, I think break statement here has no effect and could be removed.
```
for _, name := range names {
		regexp, _ := regexp.Compile(`v[1,2]/build/.*\..*`)
		name := regexp.FindString(name)
		switch name {
		case "v1/build/index.html":
			r.NewRoute().Methods(http.MethodGet).Path("/").HandlerFunc(p.index)
		case "v2/build/index.html":
			r.NewRoute().Methods(http.MethodGet).Path("/").HandlerFunc(p.indexV2)
		case ""
```